pkg/gofr/gofr.go

```diff
@@ -374,11 +351,50 @@ func (a *App) getExporter(name, host, port, url string) (sdktrace.SpanExporter,
 
 		exporter = NewExporter(url, logging.NewLogger(logging.INFO))
 	default:
-		a.container.Errorf("unsupported trace exporter: %s", name)
+		if name == "" {
+			a.container.Errorf("TRACE_EXPORTER env is missing")
+		} else {
+			a.container.Errorf("unsupported trace exporter: %s", name)
```

Comment:
```suggestion
			a.container.Errorf("unsupported TRACE_EXORTER: %s", name)
```

It would be logic because of the change you are doing when empty